src/main/java/parameterfiddler/ParameterFiddler.java

```diff
@@ -16,7 +16,7 @@
     private static String brutaltester;
     private static String refereeCommand;
     private static Bot toImprove;
-    private static ArrayList<Bot> opponents = new ArrayList<Bot>();
```

Comment:
Type erasure means `new ArrayList<X>(); `  is the same as `new ArrayList<>();`

Reply:
I know that syntax, I just don't like it that much - especially when declaration and initialization are in different lines.